.github/workflows/main.yml

```
@@ -37,40 +36,149 @@ jobs:
 
-            - name: gcc
+    gcc-test:
+        needs: build_and_publish_container
+        if: always() && (needs.build_and_publish_container.result == 'success' || needs.build_and_publish_container.result == 'skipped')
+        runs-on: ubuntu-latest
+        permissions:
+          packages: read
+          contents: read
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v2
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)